docker-compose-rule-core/src/test/java/com/palantir/docker/compose/execution/KillDownShutdownStrategyShould.java

```diff
@@ -14,14 +14,23 @@
 public class KillDownShutdownStrategyShould {
 
     @Test
-    public void call_kill_then_down() throws Exception {
+    public void stop_call_kill() throws Exception {
```

Comment:
The pattern elsewhere in this repo is that Class#method should read like a vaguely english sentence:

```
KillDownShutdownStrategyShould#call_kill_then_down (kinda expanatory)
KillDownShutdownStrategyShould#stop_call_kill (not quite as clear)
```

I'd propose naming these something like `kill_container_when_stop_method_is_called`